pkg/kubelet/rkt/rkt.go

```diff
@@ -1108,6 +1108,18 @@ func (r *Runtime) getSelinuxContext(opt *v1.SELinuxOptions) (string, error) {
 	return strings.Join(ctx, ":"), nil
 }
 
+// From the generateName or the podName return a basename for improve the logging with the Journal
```

Comment:
Also can you list an example for the generateName, podName and return value here?